frontend/packages/dev-console/integration-tests/features/topology/topology_pipeline_feature-ODC-3925

```diff
@@ -0,0 +1,31 @@
+Feature: Pipeline features in topology
+	As a user, I want to check pipeline features in topology
+
+Background:
+    Given user is at developer perspective
+    And user has installed Openshift Pipelines Operator
+    And user has selected namespace "aut-topology-pipeline"
+
+Examples:
+    | resource_type     | workload_name   |
+    | deployment        | nodejs-ex-git-1 |
+    | deployment config | dancer-ex-git-1 |
+
+
+@regression
+Scenario Outline: Sidebar for workload
```

Comment:
I think scenario description should be different [Think as per the https://issues.redhat.com/browse/ODC-3925 criteria]

Reply:
Updated